repo/packages/R/riak/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@platon see https://github.com/mesosphere/dcos-helloworld

Happy to answer any questions.